pkg/platform/kube/test/platform_test.go

```diff
@@ -40,13 +41,118 @@ import (
 	autoscalingv1 "k8s.io/api/autoscaling/v1"
 	"k8s.io/api/core/v1"
 	extensionsv1beta1 "k8s.io/api/extensions/v1beta1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 type DeployFunctionTestSuite struct {
 	KubeTestSuite
 }
 
+// Test that we get the expected brief error message on function deployment failure
+func (suite *DeployFunctionTestSuite) TestDeployFailureBriefErrorMessage() {
+	platformConfigConfigmap := suite.createPlatformConfigmapWithJSONLogger()
```

Comment:
why do you need to create it? you have direct access to the object
`suite.PlatformConfiguration`
and `suite.Controller.GetPlatformConfiguration()`

Reply:
I tried that, but I found out that changing `suite.PlatformConfiguration` won't have any effect on the processor's platformConfiguration.
The processor expects to find it on a configmap named `nuclio-platform-config`

Reply:
you have logger sink on the function spec level. look at `pkg/functionconfig/types.go:251`